src/test/java/racingcar/domain/system/GameSystemTest.java

```diff
@@ -0,0 +1,58 @@
+package racingcar.domain.system;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import racingcar.domain.car.Car;
+import racingcar.domain.carrepository.CarRepository;
+import racingcar.domain.numbergenerator.NumberGenerator;
+import racingcar.domain.result.Result;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class GameSystemTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 5, 100})
+    @DisplayName("자연수인 시도 횟수를 입력하였을 때, 객체가 생성되는지 확인")
+    void validate_test(int gameRound) {
+        GameSystem gameSystem = new GameSystem(gameRound, new NumberGenerator());
+
+        assertThat(gameSystem).isInstanceOf(GameSystemTest.class);
```

Comment:
```suggestion
        assertThat(gameSystem).isInstanceOf(GameSystem.class);
```
테스트 실패하는 부분 확인 부탁드려요!

Reply:
부끄럽습니다ㅠ

꼼꼼히 살피는 습관을 들이겠습니다ㅠㅠ

Reply:
개인적으로 팁을 드리자면 PR을 올리시기 전에 항상 전체 테스트를 돌려보시는 습관을 들이시면 좋습니다 😄